cmd/jaeger/config-remote-storage-backend.yaml

```diff
@@ -0,0 +1,65 @@
+service:
+  extensions: [jaeger_storage, jaeger_query, remote_storage, remote_storage/archive, healthcheckv2]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter]
+  telemetry:
+    resource:
+      service.name: jaeger
+    metrics:
+      level: detailed
+      readers:
+        - pull:
+            exporter:
+              prometheus:
+                host: 0.0.0.0
+                port: 8887
+    logs:
+      level: debug
+    # TODO Initialize telemetry tracer once OTEL released new feature.
+    # https://github.com/open-telemetry/opentelemetry-collector/issues/10663
+
+extensions:
+  healthcheckv2:
+    use_v2: true
+    http:
+      # use different port to avoid conflict with collector on 13133
+      endpoint: 0.0.0.0:12133
+
+  jaeger_query:
```

Comment:
why do we need query service in this setup?

Reply:
This part of the test fails without it: https://github.com/jaegertracing/jaeger/blob/main/cmd/jaeger/internal/integration/e2e_integration.go#L151-L152

Reply:
I see - we should fix that. It's using `jaeger_query` extension in order to read the storage name from it and pass that storage name to cleaner extension. But we can relax the code to also look for remote_storage extension instead of query.

Reply:
Booked #7058 to fix this